lib/msal-browser/src/controllers/NestedAppAuthController.ts

```diff
@@ -416,11 +421,7 @@ export class NestedAppAuthController implements IController {
     loginPopup(
         request?: PopupRequest | undefined // eslint-disable-line @typescript-eslint/no-unused-vars
     ): Promise<AuthenticationResult> {
-        if (request !== undefined) {
-            return this.acquireTokenInteractive(request);
-        } else {
-            throw NestedAppAuthError.createUnsupportedError();
-        }
+        return this.acquireTokenInteractive(request || DEFAULT_REQUEST);
```

Comment:
Do you track correlation-id for NAA? I think this should be:
```javascript
        const correlationId: string = request.correlationId || BrowserCrypto.createNewGuid(),
        return this.acquireTokenPopup({
            correlationId,
            ...(request || DEFAULT_REQUEST),
        });
 ```
 
Better yet, may be utilize `getRequestCorrelationId()` from `StandardController` as a common API.

Reply:
I see. Yeah, checked that you are validating in `acquireTokenInteractive`